src/components/policy/policy_external/include/policy/policy_types.h

```diff
@@ -113,7 +113,13 @@ typedef std::vector<std::string> PermissionsList;
  */
 typedef std::vector<std::string> StringArray;
 
-enum PermitResult { kRpcAllowed = 0, kRpcDisallowed, kRpcUserDisallowed };
+enum PermitResult {
+  kRpcAllowed = 0,
```

Comment:
@AKalinich-Luxoft 
I think better to rename new enum values 
kRpcParamsDisallowed --> kRpcAllParamsDisallowed
kRpcParamsUserDisallowed --> kRpcAllParamsUserDisallowed
because if only some parameters in request are disallowed, RPC can be processed and won't be disallowed

Reply:
@dev-gh good idea. Fixed in https://github.com/smartdevicelink/sdl_core/pull/1799/commits/80883f435a69895f8b0c1757f6458707c8fcfd12